src/devices/localization2DServer/Localization2DServer.h

```diff
@@ -0,0 +1,72 @@
+/*
+* Copyright (C)2017 ICub Facility - Istituto Italiano di Tecnologia
+* Author: Marco Randazzo
+* email:  marco.randazzo@iit.it
+* website: www.robotcub.org
+* Permission is granted to copy, distribute, and/or modify this program
+* under the terms of the GNU General Public License, version 2 or any
+* later version published by the Free Software Foundation.
+*
+* A copy of the license can be found at
+* http://www.robotcub.org/icub/license/gpl.txt
+*
+* This program is distributed in the hope that it will be useful, but
+* WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General
+* Public License for more details
+*/
```

Comment:
This copyright is wrong. We cannot accept GPL licensed code in YARP.
Also this is not the right header for GPL files in YARP.

Reply:
fixed